addOns/pscanrulesBeta/src/main/resources/org/zaproxy/zap/extension/pscanrulesBeta/resources/Messages.properties

```diff
@@ -26,7 +26,7 @@ pscanbeta.directorybrowsing.extrainfo={0}
 pscanbeta.hashdisclosure.name=Hash Disclosure
 pscanbeta.hashdisclosure.desc=A hash was disclosed by the web server
 pscanbeta.hashdisclosure.soln=Ensure that hashes that are used to protect credentials or other resources are not leaked by the web server or database. There is typically no requirement for password hashes to be accessible to the web browser.      
-pscanbeta.hashdisclosure.refs=https://www.owasp.org/index.php/Top_10_2013-A6-Sensitive_Data_Exposure\nhttp://projects.webappsec.org/w/page/13246936/Information%20Leakage\nhttp://openwall.info/wiki/john/sample-hashes
+pscanbeta.hashdisclosure.refs=https://wiki.owasp.org/index.php/Top_10_2013-A6-Sensitive_Data_Exposure\nhttp://projects.webappsec.org/w/page/13246936/Information%20Leakage\nhttp://openwall.info/wiki/john/sample-hashes
```

Comment:
Do we really want to link to the old OWASP site? https://wiki.owasp.org/index.php/Top_10_2013-A6-Sensitive_Data_Exposure doesnt look good (and other wiki.owasp.org links)

Reply:
I'm good with dropping them.